com.usebottles.bottles.yml

```diff
@@ -38,6 +38,7 @@ finish-args:
 - --filesystem=~/.local/share/bottles:create
 - --device=all
 - --filesystem=host
```

Comment:
Shouldn't this be enough?

Reply:
```
[📦 com.usebottles.bottles ~]$ LC_ALL=C ls /media
ls: cannot access '/media': No such file or directory
```
Unfortunately no.

Reply:
Seems this is a bug in flatpak, as it should share /media with `host` permission. https://github.com/flatpak/flatpak/blob/1.10.1/doc/flatpak-metadata.xml#L223-L234

Reply:
Reported https://github.com/flatpak/flatpak/issues/4163

Reply:
I've reinstall bottles and now /media is visible, strange.